addons/website_sale/models/product_template.py

```diff
@@ -133,6 +139,11 @@ def _default_website_sequence(self):
 
     #=== COMPUTE METHODS ===#
 
+    @api.depends('is_published')
+    def _compute_published_date(self):
+        for template in self:
+            template.published_date = template.is_published and fields.Datetime.now()
```

Comment:
This will set `False` as the value for unpublished products, which seems odd to me. I think we should aim to either:
- update the date only when a product is (re-)published
- **or** every time it's unpublished/republished.

Reply:
Changed it to the first strategy: updating `published_date` whenever it's (re-)published.